tools/clang/test/DXC/Inputs/include-main.hlsl

```diff
@@ -0,0 +1,13 @@
+// Test display include process with /Vi
```

Comment:
Is this an input or a test? If it is a test, it shouldn't be in the Inputs subdirectory. Lit doesn't traverse into directories named `inputs` to identify test cases.